CRM/Custom/Form/Field.php

```diff
@@ -950,6 +950,17 @@ public function postProcess() {
       $params['is_search_range'] = 0;
     }
 
+    // Serialization cannot be changed on update
+    if ($this->_id) {
+      unset($params['serialize']);
+    }
+    elseif (in_array($params['html_type'], ['Select', 'Country', 'StateProvince'])) {
```

Comment:
Shouldn't 
Country -> Select Country
StateProvince -> Select State/Province 

As per https://github.com/civicrm/civicrm-core/pull/16992/files#diff-cac3f19d5b4be8233ee77c257538bc40R163?

Reply:
You would think so, but this form is very strange. I still haven't figured out why it works at all.

Reply:
Ok I stepped through this with a debugger and you're right. I've updated the commit to check for any string starting with "Select".